scripts/lint.sh

```diff
@@ -15,14 +15,15 @@ PASSED=true
 
 OS="$(uname)"
 
- collect the files on MacOS
+ # collect the files on MacOS
  if [[ "$OS" == "Darwin" ]]; then
      FILES=$(find -E . -type f -regex '\./(gator|tests)/.*.py')
  else
     FILES=$(find . -type f -regextype posix-extended -regex '\./(gator|tests)/.*.py')
  fi
 
 # lint all of the Python source code files
+ FILE_FOLDERS=("src" "tests")
  FILES="$FILES *.py"
```

Comment:
@stephensonc I think the reason why the `$FILES` are not correctly passing into the linters is that it is trying to find `gator` or `tests` here in line 20 and line 22

Reply:
The strange thing here is that Black actually finds the files properly using that regex. That is why I decided that it would be safer to have a separate folder variable.

Reply:
I think black works probably because it just checking if the codes are correctly formatted, and we happened to have `tests` and they happened to be correctly formatted. Maybe try to run an echo function to see if the `$FILES` being found is actually right. What you have here can also work, it is basically the same thing, but I doubt if black is actually checking the files in `src`

Reply:
Black already searches for exclusively python files already, right? There should be no need to list all of the files in that case, and then the folder list I created can be used for all of the linters, and the specific list of files likely cannot (though it definitely isn't parsing correctly because it is passing "*.py" as an actual file)

Reply:
Yes. What I meant by running the echo function is just to see whether `src` is being correctly passed in for now. The `$FILE_FOLDERS` also works